<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="202.954" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2023-03-27T14:19:57" tests="0" file="cypress/e2e/profiles/profile.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Profile" timestamp="2023-03-27T14:19:57" tests="1" time="202.946" failures="1">
    <testcase name="Profile &quot;before each&quot; hook for &quot;Test that user is able to navigate to the profile page&quot;" time="30.274" classname="&quot;before each&quot; hook for &quot;Test that user is able to navigate to the profile page&quot;">
      <failure message="`cy.visit()` failed trying to load:

https://devcardholder.essolo.com/onboarding/signin

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  &gt; Error: ESOCKETTIMEDOUT

Common situations why this would fail:
  - you don&apos;t have internet access
  - you forgot to run / boot your web server
  - your web server isn&apos;t accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Profile`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

https://devcardholder.essolo.com/onboarding/signin

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: ESOCKETTIMEDOUT

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Profile`
    at <unknown> (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:138252:82)
    at visitFailedByErr (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:137644:12)
    at <unknown> (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:138251:11)
    at tryCatcher (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:8914:23)
    at Promise._settlePromiseFromHandler (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:6849:31)
    at Promise._settlePromise (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:6906:18)
    at Promise._settlePromise0 (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:6951:10)
    at Promise._settlePromises (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:7027:18)
    at _drainQueueStep (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:3621:12)
    at _drainQueue (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:3614:9)
    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:3630:5)
    at Async.drainQueues (https://devcardholder.essolo.com/__cypress/runner/cypress_runner.js:3500:14)
From Your Spec Code:
    at LoginPage.accessLoginModal (webpack:///./cypress/support/pages/shared/login-page.js:9:11)
    at Context.eval (webpack:///./cypress/e2e/profiles/profile.cy.js:11:16)

From Node.js Internals:
  Error: ESOCKETTIMEDOUT
      at ClientRequest.<anonymous> (<embedded>:1955:73317)
      at Object.onceWrapper (node:events:641:28)
      at ClientRequest.emit (node:events:527:28)
      at TLSSocket.emitRequestTimeout (node:_http_client:771:9)
      at Object.onceWrapper (node:events:641:28)
      at TLSSocket.emit (node:events:539:35)
      at Socket._onTimeout (node:net:516:8)
      at listOnTimeout (node:internal/timers:559:17)
      at process.processTimers (node:internal/timers:502:7)]]></failure>
    </testcase>
  </testsuite>
</testsuites>